pkg/image/image.go

```diff
@@ -59,3 +72,137 @@ func ParseReference(ref string) (imagesource.TypedImageReference, error) {
 	}
 	return imagesource.TypedImageReference{Ref: dst, Type: dstType}, nil
 }
+
+/* GetConfigDirFromOCI verifies the ref is for an image of type OCI
+ * It then unpacks the layers, searches for the configs folder within
+ * the unpacked content and returns it
+ */
+func GetConfigDirFromOCICatalog(ctx context.Context, ref string) (string, error) {
+
+	ociImgSrc, path, err := getOCIImgSrcFromPath(ctx, ref)
+	if err != nil {
+		return "", err
+	}
+	defer ociImgSrc.Close()
+
+	manifest, err := getManifest(ctx, ociImgSrc)
+	if err != nil {
+		return "", err
+	}
+
+	untarredLayer := ""
+	for _, layer := range manifest.LayerInfos() {
+		if !layer.EmptyLayer {
+			tmpDir := "/tmp/oc-mirror/"
```

Comment:
I haven't checked yet but as far as I recall, we can get temp directory automatically using go libraries instead using a static one?